core/build.gradle.kts

```diff
@@ -16,6 +16,9 @@
  */
 import com.github.vlsi.gradle.crlf.CrLfSpec
 import com.github.vlsi.gradle.crlf.LineEndings
+import com.github.vlsi.gradle.ide.dsl.settings
+import com.github.vlsi.gradle.ide.dsl.taskTriggers
+import org.jetbrains.kotlin.backend.common.onlyIf
```

Comment:
this `onlyIf` looks weird

Reply:
Not clear if you're asking for a change. It works and was the way I found others doing this. Keep in mind that this is my first effort in gradle and kotlin.

Reply:
Please drop `onlyIf` import. The package name `org.jetbrains.kotlin.backend.common` looks as if it has something to do with `kotlin backend` which is not something you want to use here. I belive the import has been added by mistake. You should call `Task#onlyIf` method which should not require an import.

Reply:
You were right. IDE (not)helpfulness. Fixed.